scripts/main.js

```diff
@@ -0,0 +1,57 @@
+import * as handlers from './handlers.js';
+import * as elements from './elements.js';
+import { loadQuestions, loadTestimonials } from './utils.js';
+
+// Adding events
+elements.mobileBtn.addEventListener('click', handlers.handleMobileBtnClick);
+
+elements.teamMembers.forEach(teamMember => {
+  teamMember.addEventListener('mouseenter', handlers.handleTeamMemberEnter);
+  teamMember.addEventListener('mouseleave', handlers.handleTeamMemberOut);
+});
+
+elements.portafolioImages.forEach(portafolioImage => {
+  portafolioImage.addEventListener(
+    'mouseenter',
+    handlers.handlePortafolioImageEnter
+  );
+  portafolioImage.addEventListener(
+    'mouseleave',
+    handlers.handlePortafolioImageOut
+  );
+});
+
+elements.questions.forEach(question => {
+  question.addEventListener('click', handlers.handleQuestionClick);
+});
+
+elements.testimonialButtons.forEach(button => {
+  button.addEventListener('click', handlers.handleTestimonialButtonClick);
+});
+
+window.addEventListener('resize', () => {
+  loadQuestions(elements);
+});
```

Comment:
avoid resize events, they trigger a ton of times

Reply:
In this case I need it, because of my implementation for the question animations, when the user change the size of the window, I need to recalculate the size of the questions.

Reply:
check if there is a way to do it with css and not calculating the height

Reply:
Yes, there are another ways, but have some drawbacks. I wanted the effect I get with this implementation. I could also put a static height into a class, but if the question is more height than that, I have to used the scroll, which is not so bad, but I don't wanted that effect, so I came out with this solution. If you prefer that, I could make it in that way.